{"name":"LoginCheck","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"expected [ContactsNew] but found [Contacts]","trace":"java.lang.AssertionError: expected [ContactsNew] but found [Contacts]\r\n\tat org.testng.Assert.fail(Assert.java:96)\r\n\tat org.testng.Assert.failNotEquals(Assert.java:776)\r\n\tat org.testng.Assert.assertEqualsImpl(Assert.java:137)\r\n\tat org.testng.Assert.assertEquals(Assert.java:118)\r\n\tat org.testng.Assert.assertEquals(Assert.java:453)\r\n\tat org.testng.Assert.assertEquals(Assert.java:463)\r\n\tat com.qa.TestClass.TestClass.LoginCheck(TestClass.java:55)\r\n\tat sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\r\n\tat sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)\r\n\tat sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)\r\n\tat java.lang.reflect.Method.invoke(Unknown Source)\r\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)\r\n\tat org.testng.internal.Invoker.invokeMethod(Invoker.java:583)\r\n\tat org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)\r\n\tat org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)\r\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)\r\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:648)\r\n\tat org.testng.TestRunner.run(TestRunner.java:505)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:455)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:364)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1137)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1049)\r\n\tat org.testng.TestNG.run(TestNG.java:1017)\r\n\tat org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)\r\n\tat org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)\r\n\tat org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)\r\n"},"stage":"finished","steps":[{"name":"login with username:com.qa.ElementUtils.CredentialsManager@36bed37a and password{1}","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[{"name":"c1","value":"com.qa.ElementUtils.CredentialsManager@36bed37a"}],"start":1597419908329,"stop":1597420155135}],"attachments":[{"name":"Page screenshot","source":"3300f22c-cfab-44e2-b99d-f4e40760e6fc-attachment","type":"image/png"},{"name":"LoginCheck failed and screenshot taken!","source":"11185103-aade-4762-8925-8f8571370239-attachment","type":"text/plain"}],"parameters":[],"start":1597419908198,"stop":1597420157506,"uuid":"bfe52f32-421b-48a5-a63c-6a5dce1fa70d","historyId":"b7d49406b2e446be95b433ed10650955","fullName":"com.qa.TestClass.TestClass.LoginCheck","labels":[{"name":"package","value":"com.qa.TestClass.TestClass"},{"name":"testClass","value":"com.qa.TestClass.TestClass"},{"name":"testMethod","value":"LoginCheck"},{"name":"parentSuite","value":"TEST RUNNERS"},{"name":"suite","value":"Functional Regression Test"},{"name":"subSuite","value":"com.qa.TestClass.TestClass"},{"name":"host","value":"DESKTOP-2LONNQF"},{"name":"thread","value":"10068@DESKTOP-2LONNQF.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"},{"name":"story","value":"Login Page Tittle"},{"name":"severity","value":"critical"}],"links":[]}